### PR TITLE
PUBDEV-3465: Sync up functionality of download_mojo and download_pojo in R & Py

### PR DESCRIPTION
**Pull request by navdeep-G**
This PR will do the following:
1. Print the POJO file name if `path != ""` otherwise it will pretty print the POJO.
2. Print the MOJO file name ,i.e., `GBM1234.zip`

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/269)

### CODE REVIEW

**Comment by ledell** [comment]
I think it should always print the POJO file name, regardless of the path.  I thought that's what we decided so that POJO and MOJO will function the same way?

**Comment by navdeep-G** [comment]
If `path=""`, then the file is not saved anywhere. Thus, the name is not very useful.

**Comment by navdeep-G** [comment]
Okay to merge?

File: h2o-r/h2o-package/R/kvstore.R [hunk 1]
```diff
@@ -252,8 +252,9 @@ h2o.download_pojo <- function(model, path="", getjar=NULL, get_jar=TRUE) {
   pojoname = gsub("[+\\-* !@#$%^&()={}\\[\\]|;:'\"<>,.?/]","_",model_id,perl=T)
   
   file.path <- paste0(path, "/", pojoname, ".java")
-  if( path == "" ) cat(java)
-  else {
+  if( path == "" ){
```
Comment (mmalohlava) [inline h2o-r/h2o-package/R/kvstore.R]: Just curious question, why is default `""` instead of `NULL`.

Comment (navdeep-G) [inline h2o-r/h2o-package/R/kvstore.R]: Not sure (it has been like this for some time...). But I guess it could be `NULL`

Comment (mmalohlava) [inline h2o-r/h2o-package/R/kvstore.R]: It is just minor comment, if i look at method definition, the first arg default is `""`, the second one is `NULL`.

Comment (ledell) [inline h2o-r/h2o-package/R/kvstore.R]: Yeah, that's atypical.  It should probably be `NULL`.  Should we change this now, or open a new JIRA?

Comment (navdeep-G) [inline h2o-r/h2o-package/R/kvstore.R]: I can just change it now.

**Comment by navdeep-G** [comment]
Just updated to set `path=NULL` by default. Running tests now in PR system.

File: h2o-r/h2o-package/R/kvstore.R (hunk 1 above)
Comment (ledell) [inline h2o-r/h2o-package/R/kvstore.R]: If it's not already, we should also have `h2o.download_mojo` path arg default to `NULL` as well.

Comment (navdeep-G) [inline h2o-r/h2o-package/R/kvstore.R]: Default path is the current working directory.

Comment (navdeep-G) [inline h2o-r/h2o-package/R/kvstore.R]: POJO in Python dumps output to the screen if `path` is not specified.

**Comment by navdeep-G** [comment]
Tests failed, but its a false positive. Data could not be found...

**Pull request merged**